build_linux.sh

```diff
@@ -46,5 +46,5 @@ export LD_LIBRARY_PATH=$LPCNETDIR/build_linux/src
 # Finally, build freedv-gui
 cd $FREEDVGUIDIR && git pull
 mkdir  -p build_linux && cd build_linux && rm -Rf *
-cmake -DCMAKE_BUILD_TYPE=Debug -DCODEC2_BUILD_DIR=$CODEC2DIR/build_linux -DLPCNET_BUILD_DIR=$LPCNETDIR/build_linux ..
+cmake -DUSE_PULSEAUDIO=1 -DCMAKE_BUILD_TYPE=Debug -DCODEC2_BUILD_DIR=$CODEC2DIR/build_linux -DLPCNET_BUILD_DIR=$LPCNETDIR/build_linux ..
```

Comment:
So thinking about merge ... are we intending to mak Pulse the default?

Reply:
Should we be building both PortAudio and Pulse versions when we push PRs to ensure both versions build cleanly?  This could perhaps be managed by adding a CLI option to `build_linux.sh` and calling it twice in `cmake.yml`

Reply:
The intention was to make it default on Linux given that PulseAudio seems to be the "standard" way sound is done (soon to be pipewire). That said, it would be a good idea to build both on PR push just to be sure compiles don't break.

Reply:
Per below, the default is currently PortAudio. Both are building on PR push now, however.